app/client/platforms/bedrock.ts

```
@@ -0,0 +1,800 @@
+  }
+
+  async models() {
+    return [];
+  }
+}
+
+function bedrockStream(
+  chatPath: string,
+  requestPayload: any,
+  headers: any,
+  funcs: Record<string, Function>,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)